src/Microsoft.ApplicationInsights/Extensibility/Implementation/Endpoints/Constants.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Endpoints
+{
+    /// <summary>
+    /// Endpoint Constants.
+    /// </summary>
+    internal static class Constants
+    {
+        /// <summary>Default endpoint for Breeze (aka Ingestion).</summary>
+        internal const string BreezeEndpoint = "https://dc.services.visualstudio.com/";
```

Comment:
We probably want to call it "IngestionEndpoint" as Breeze is the code name.
No strong opinion though, up to you.